File: JavaHarry/factorial.java
public class factorial{
	static int factorial(int n){
		if(n==0 || n==1){
		    return 1;
		}
		return n*factorial(n-1);
	}
	public static void main(String [] args){
		int fact = factorial(5);
		System.out.println(fact);
	}
}

File: Codechef using C-C++/factorial.c
#include<stdio.h>

int main()
{
	   int i,n,fact=1;


    printf("enter n : ");
    scanf("%d",&n);
    
    for(i=1; i<=n; i++)
    {
    
    fact= fact * i ;
    
    }
    printf("fact of %d is %d",n,fact);
    
    
	return 0;
}